src/profile-logic/gecko-profile-versioning.js

```diff
@@ -861,5 +861,96 @@ const _upgraders = {
     }
     convertToVersion19Recursive(profile);
   },
+  [20]: profile => {
+    // The idea of phased markers was added to profiles. This upgrader removes the `time`
+    // field from markers and replaces it with startTime, endTime and phase.
+    type OldSchema = {| name: 0, time: 1, category: 2, data: 3 |};
+    type Payload = $Shape<{
+      startTime: number,
+      endTime: number,
+      type: string,
+      interval: string,
+    }>;
+
+    const INSTANT = 0;
+    const INTERVAL = 1;
+    const INTERVAL_START = 2;
+    const INTERVAL_END = 3;
+
+    // Profiles now have an innerWindowID property in the frameTable.
```

Comment:
I don't think this comment applies for this upgrader :)

Reply:
COPY PASTA

![COPY PASTA](https://media.giphy.com/media/13AJJwFqUsdoWY/giphy.gif)